src/libraries/Extensions/Microsoft.Agents.Extensions.Teams.AI/StreamingChannelData.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
```

Comment:
Do we need this file now? 
This puts stream data inside channel data,  which is not where it goes any longer. 
Is there anything missing from the core stream data concept we need to move over?